### PR TITLE
Create Layout

### LINKED ISSUE #3: Layout
Get the layout initialized

### PR DESCRIPTION
**Pull request by krferrell**
## Changes
1. Setup the layout for the landing page

## Purpose
To create a canvas to start adding information to

## Approach
Used React Slots Pattern to render the layout

Closes #3

### CODE REVIEW

**Review by BlackCubes — APPROVED** [review]
Very good!

File: the-toad-tribune/src/layout/Ads.tsx [hunk 1]
```diff
@@ -0,0 +1,5 @@
+const Ads = () => {
+  return <>Ads</>;
```
Comment (Lap343) [inline the-toad-tribune/src/layout/Ads.tsx:2]: Ads? Getting paid?

File: the-toad-tribune/src/layout/Layout.tsx [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+  return (
+    <>
+      {Ads}
+      {Animals}
+      {MainArticle}
+      {Movies}
+      {Navigation}
+      {Politics}
+      {Sports}
+      {Stonks}
```
Comment (Lap343) [inline the-toad-tribune/src/layout/Layout.tsx:35]: lol

File: the-toad-tribune/src/SubApp.tsx [hunk 3]
```diff
@@ -0,0 +1,32 @@
+import {
+  Ads,
+  Animals,
+  Layout,
+  MainArticle,
+  Movies,
+  Navigation,
+  Politics,
+  Sports,
+  Stonks,
```
Comment (Gjhnsn) [inline the-toad-tribune/src/SubApp.tsx:10]: $tonks

File: the-toad-tribune/src/layout/Ads.tsx (hunk 1 above)
Comment (krferrell) [reply the-toad-tribune/src/layout/Ads.tsx:2]: Always

**Review by Lap343 — APPROVED** [review]
Nice start

**Review by Gjhnsn — APPROVED** [review]
Good clean looking setup for the project

**Review by JorgeLVilla — APPROVED** [review]
Sick app name lolss

File: the-toad-tribune/src/SubApp.tsx (hunk 3 above)
Comment (JorgeLVilla) [reply the-toad-tribune/src/SubApp.tsx:10]: Big money big money

File: the-toad-tribune/src/layout/Ads.tsx (hunk 1 above)
Comment (JorgeLVilla) [reply the-toad-tribune/src/layout/Ads.tsx:2]: Making money while you sleep

**Pull request merged**